src/disco/components/Addon/index.js

```diff
@@ -45,9 +45,9 @@ const CSS_TRANSITION_TIMEOUT = { enter: 700, exit: 300 };
 
 export const installStaticTheme = async (event, props) => {
   event.preventDefault();
-  const { status } = props;
+  const { hasAddonManager, status } = props;
 
-  if (status === UNINSTALLED) {
+  if (hasAddonManager && status === UNINSTALLED) {
```

Comment:
ah yes, I think we should not have to call this method if we have no mozAddonManager, but it should not be done here IMO. Can we prevent the call where we bind the onClick event?

Reply:
i thought about doing that but then there will still be a link where we want to do event.preventDefault...

unless I make it not a link with this condition...is that better?